test/Riok.Mapperly.Tests/Mapping/EnumerableTest.cs

```diff
@@ -633,6 +759,78 @@ public Task MapToExistingQueueShouldWork()
         return TestHelper.VerifyGenerator(source);
     }
 
+    [Fact]
+    public Task EnumerableToReadonlyImmutableList()
```

Comment:
Not sure if this and the following tests make sense.... They are trying to map from an `int` property to an `Immutable*`, shouldn't the source property type be of type `IEnumerable<int>`?